3001-4000/LIT3326Melchiz1.xml

```diff
@@ -30,6 +31,20 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange unit="item">5280</citedRange>
                </bibl>
             </listBibl>
+            <listWit>
+               <witness corresp="BNFet146" xml:id="P"/>
```

Comment:
what are these xml:id for? unless you are producing below a critical edition with an apparatus referring to these sigla these are not needed

Reply:
They are probably from Weischer's edition, are they? If they come from an accepted edition, I think, it make sense to keep them. However, they should be encoded in a way, that they are visible on the website, e.g.:
`<witness corresp="BNFet146">Paris, Bibliothèque nationale de France, BnF Éthiopien 146 (ms. P in Weischer's edition)</witness>`

Here you can also note the Matenadaran fragment:
`<witness>Erevan, Matenadaran, Fragment Nr. 947 (Z2 in Weischer's edition)</witness>`
or perhaps:
`<witness>Erevan, Matenadaran Library, Fragment Nr. 947 (Z2 in Weischer's edition)</witness>`

@eu-genia: Is it possible to create a folder for Erevan and a stub for this fragment?